.vib/mediawiki/vib-verify.json

```diff
@@ -17,12 +17,52 @@
       ]
     },
     "verify": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/mediawiki"
+        },
+        "runtime_parameters": "bWVkaWF3aWtpVXNlcjogdXNlcgptZWRpYXdpa2lQYXNzd29yZDogQ29tcGxpY2F0ZWQxMjMhNAptZWRpYXdpa2lFbWFpbDogdXNlckBleGFtcGxlLmNvbQptZWRpYXdpa2lIb3N0OiBiaXRuYW1pLW1lZGlhd2lraS5teQpzZXJ2aWNlOgogIHR5cGU6IExvYWRCYWxhbmNlcgogIHBvcnRzOgogICAgaHR0cDogODA=",
+        "target_platform": {
+          "target_platform_id": "{VIB_ENV_TARGET_PLATFORM}",
+          "size": {
+            "name": "S4"
+          }
+        }
+      },
       "actions": [
         {
           "action_id": "trivy",
           "params": {
             "threshold": "CRITICAL",
-            "vuln_type": ["OS"]
+            "vuln_type": [
+              "OS"
+            ]
+          }
+        },
+        {
+          "action_id": "goss",
+          "params": {
+            "resources": {
+              "path": "/.vib/mediawiki/goss"
+            },
+            "remote": {
+              "workload": "deploy-mediawiki"
+            }
+          }
+        },
+        {
+          "action_id": "cypress",
+          "params": {
+            "resources": {
+              "path": "/.vib/mediawiki/cypress"
+            },
+            "endpoint": "lb-mediawiki-http",
+            "app_protocol": "HTTP",
+            "env": {
+              "username": "user",
+              "password": "Complicated123!4"
```

Comment:
Will this app allow for the standard user/pass?

```
    "username": "user",
    "password": "ComplicatedPassword123!4"
```

Reply:
It does, my bad! I missed changing this when I started the tests. Fixed now